### ISSUE TITLE
House ownership disappeared

### ISSUE THREAD

**Issue by phoenixflame611** [jabberzac/Overthrow#99]
Currently running a dedicated server, had to shutdown to update the mod and we all lost ownership for our houses we spawned into. Our gear/safes are still there and it's still marked with a "house" icon on the map, but we cannot place items or fast travel to the.  0.7.2.0.

**Comment by shaggnater** [comment]
Having the same issue...

**Comment by MysticDaedra** [comment]
I am having either the same issue or a related issue... or an extension of the issue. Loaded my persistent save, and it seems my marked "home" has moved... i.e., the "home" icon on the map, as well as my spawn has moved to a different shack. This shack is marked as owned, but my original shack (which still has inventory plus map and other starting furniture) is no longer marked as owned.

**Comment by KlownGT** [comment]
My group has had this issue. We first hosted on a members PC and two of had the same spawn house, with only 1 owner. Now that we are dedicated everyone got their own house, although mine seems to be owned by no one after server restart. Says "Owned by  *blank*".

**Comment by KlownGT** [comment]
This probably has something to do with fast travel issue we are having as well. Even when in a group we can't fast travel to each other's locations, which when it comes to Greenfor resources like police stations and warehouses, my opinion should all be faction travels rather then ownership. Again is COOP is the point there shouldn't be travel limits, they have passcodes on safes as Is lol

**Comment by geekwithbadge** [comment]
We are having the same issue.

**Comment by shaggnater** [comment]
I agree with KlownGT... This is a Co-op game and I think everyone logged in to the server should be set into the same group automatically, thus possibly making it easier to fix the "owner" issue with saves & restarts.

**Comment by chrisoliver1** [comment]
Having this problem currently on a dedicated server. Very annoying as no one from the group can fast travel to their houses. A fix or solution would be a life saver right now.
